invite/forms.py

```diff
@@ -12,17 +11,15 @@ def validate_username(value):
 
 
 def validate_user_email(value):
-    insensitive_emails = (
-        [e.lower() for e in User.objects.all().values_list('email', flat=True)]
-    )
-    assert False, insensitive_emails
-    if value.lower() not in insensitive_emails:
-        raise ValidationError('The email provided doesn\'t belong to any user')
+    if User.objects.filter(email__iexact=value).exists():
```

Comment:
If we keep this to use in `validataors` of the `IForgotForm`, I think this needs to change to:
```suggestion
    if not User.objects.filter(email__iexact=value).exists():
```